Dockerfile

```diff
@@ -87,6 +87,9 @@ RUN wget https://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types
 # Copy entrypoint
 COPY ./docker/files/usr/local/bin/entrypoint /usr/local/bin/entrypoint
 
+# Copy configuration
+VOLUME [ "/configuration" ]
```

Comment:
Is it correct, the configuration is not to the root anymore ?

Reply:
Yes we can remove it indeed